dwm1001/.cargo/config.toml

```diff
@@ -0,0 +1,16 @@
+[build]
+target = "thumbv7em-none-eabihf"
+
+[target.thumbv7em-none-eabihf]
+runner = "probe-run --chip nRF52832_xxAA"
+rustflags = [
+    "-C",
+    "linker=flip-link",
+    "-C",
+    "link-arg=-Tlink.x",
+    "-C",
+    "link-arg=-Tdefmt.x",
+]
```

Comment:
To add defmt support to the examples, we need to update the runner. I switched from `config` to `config.toml` (which I believe is supported, but being phased out?) to get better IDE support.

Reply:
:+1: